src/index.js

```diff
@@ -0,0 +1,68 @@
+import './style.css';
+import 'bootstrap';
+
+let todoTasks = [
+  {
+    description: 'Do the dishes / Sample element',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Do the laundry / Sample element',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Walk the dog / Sample element',
+    completed: false,
+    index: 2,
+  },
+];
+
+// elements
+
+// Selectors
+const todoInput = document.querySelector('.todoInput');
+
+// Event Listeners
+
+// Functions
+function searchID(id) {
+  return document.getElementById(id);
+}
+
+function fillOutList(list) {
+  list.sort((a, b) => a.index - b.index);
+
+  const listWrapper = searchID('listWrapper');
+
+  const listElement = document.createElement('li');
+  listElement.classList.add('list-group-item', 'pl-5', 'pt-4', 'pb-4', 'clearfix');
+
+  const checkBox = document.createElement('input');
+  checkBox.type = 'checkbox';
+  checkBox.classList.add('form-check-input', 'float-left');
+  listElement.appendChild(checkBox);
+
+  const listText = document.createElement('span');
+  listText.innerText = todoInput.value;
+  listElement.appendChild(listText);
+
+  const dragIcon = document.createElement('span');
+  dragIcon.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+  dragIcon.classList.add('float-right');
+  listElement.appendChild(dragIcon);
+
+  listWrapper.appendChild(listElement);
+
```

Comment:
- [ ] This was good having a function that is responsible for creating a list item(Task) with its content, but it seems to be rendered to the DOM only once, whereas remembering that the number of rendered items should be the same as what we have in the array. which means we actually need an iterator or a loop that will iterate over the array and dynamically render each element to the DOM as what is said in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/55465133/134430856-eba32954-725f-492d-b3d7-0df13f249cef.png)

Please let's add the logic of rendering the list of tasks to the dom dynamically using an iterator 😉